### PR TITLE
Added a square root function

### PR DESCRIPTION
**Pull request by maxrobot**
Also would be good to know if anyone has opinions on the root methodology?

We could add in a break for the loop on line 70 with a tolerance.

Also removed the generic CosmWasm notices from cargo generate.

### CODE REVIEW

**Review by nicolasbaum — APPROVED** [review]
Looks ok

File: packages/injective-math/src/fp_decimal/exp.rs [hunk 1]
```diff
@@ -51,6 +51,29 @@ impl FPDecimal {
         }
         val
     }
+
+    // a^(0.5)
+    pub fn _root(a: FPDecimal) -> Option<FPDecimal> {
```
Comment (gorgos) [inline packages/injective-math/src/fp_decimal/exp.rs]: if it's a function meant for external use, you should call it `root`. And maybe `sqrt` is better?

**Review by gorgos — CHANGES_REQUESTED** [review]
`sdk.Dec` is using max 100 iterations https://github.com/cosmos/cosmos-sdk/blob/44495e7a7934f00fe289c9f059ff3dba6582b43e/math/dec.go#L423, but Im also fine with the current approach.

**Comment by DrunkRandomWalker** [comment]
One suggestion set `exp(0.5)` to use `sqrt`

**Review by DrunkenRandomWalker — APPROVED** [review]
LGTM, One suggestion: set `exp(0.5)` to use `sqrt`

**Review by gorgos — APPROVED** [review]

**Pull request merged**